RLTest/__main__.py

```diff
@@ -404,28 +405,39 @@ def _runTest(self, test, numberOfAssertionFailed=0, prefix=''):
                 return 0
 
             fn = lambda: test.target(env)
+            before = (lambda: before(env)) if before else None
```

Comment:
why we need to set the before in lambda? why not just calling before directly?

Reply:
it wraps the lambda so that we don't need to know if it needs the argument

Reply:
Why not just passing the argument anyway ?